### ISSUE TITLE
quest not working

### ISSUE THREAD

**Issue by JustKruzin** [dalaranwow/dalaran-wow#4933]
[//]: # (daily quest , they grew up fast, unable to hand in for reward
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**behaviour**: daily quest not working, spent 20 days completing. quest is they grow up fast.
**Expected behaviour**: You should receive reward whistle of the venomhide ravensaur.
Yellow question mark lit up on map, but greyed out on quest giver when trying to hand in

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @JustKruzin, for future reports please fill the entire template so we get all the needed information for the issue you are reporting. Thank you.
Regarding your issue just abandon and retake the quest. It's a general issue not just specific to this quest and it's reported on the bug tracker.
